### DISCUSSION TITLE
Pull shark achievement should work for repositories I don't own only

### DISCUSSION THREAD

**Discussion by lubiah** [community/community#18243] [Other Feature Feedback, Questions, & Ideas]
I just created a branch on one of my repositories to make an update, after merging it, I realized the pull shark achievement counter increased. I retried it again and realized the counter just kept on increasing.

The point is, this doesn't actually make the achievement feel real enough, we will always merge our own pull requests but not everyone will merge our pull request and that's the point.
If someone merges our pull request, then it means it worthy to them hence it becomes an achievement for us

**Comment by MrKrishnaAgarwal** [comment]
True,
Some people create commits in their own repositories with pull requests.
I also follow this.

**Comment by FilipeChagasDev** [comment]
I also received the Pull Shark achievement for a pull request I made in a repository I manage.  I made this pull request and merged it on my own repository to learn how pull requests work on github. It makes no sense that I get an achievement for this.

**Reply by lubiah** [reply]
Yeah, very true

**Reply by hsleonis** [reply]
Agreed.

**Comment by trackpadpro** [comment]
I fully agree. Also, though, how does pull shark even count anything? Is it telling me that I have only had 2 PRs merged? The most I have seen anyone have yet is bronze at 2, but it doesn't make sense, because I have definitely opened far more in other repositories. I feel like these achievements should encourage people to contribute to open source projects that aren't their own, but the only 2 PRs I get credit for are one in my own repo and one in a repo that an org I am a member of owns. 

EDIT: Okay, fine, I figured out what it means by x2. Just second stage, and you have giant gaps between stages in an exponential fashion. That makes it a little better, I guess, but I still believe it should be pull requests that you could not merge yourself. It would be a much better badge if it instead encouraged contributing to different repos. Even that wouldn't even be fantastic because you could be encouraging people who work on a labor of love to instead ditch something they've contributed a great deal to.

**Reply by MrKrishnaAgarwal** [reply]
Agree👍

**Reply by Kage0923** [reply]
Agree

**Comment by Nukesor** [comment]
On the other hand, some people own repositories and contribute to them via PRs for non-personal features and issues.

For instance, I regularily implement community-requested features for my projects without needing them myself. Having to create a separate organization just to get aknowledgment for those PRs would kind of miss the point.

**Reply by alexkasongo** [reply]
This.

**Reply by ghadeeras** [reply]
I would agree if the PR is associated to an issue someone else opened?

**Comment by tsarthur** [comment]
Yep

**Comment by p-arch-paul** [comment]
My sandboxes cannot open and checkers are also failing

**Comment by epistax1s** [comment]
It seems to me that you should create 2 badges, one to give for a contribution to your repositories, the second for a contribution to someone else's.

**Reply by ghadeeras** [reply]
Indeed, one for contribution, and one for openness/inclusivity.

**Comment by p-arch-paul** [comment]
Cam you just show how to Make icons. screenshot them

**Comment by KOLANICH** [comment]
I think pull shark achievement should be removed at all. Giving achievements for just doing the things the platform is intended for is just ridiculous. Having an achievement for sending less than 10 prs is extremely ridiculous. I think this achievement should be removed and instead a **small** and simple counter of cumulative (in all the repos and orgs) prs, issues, commits, repos, stars, follows, forks and dependent packages should be added. Just statistics, no achievement for this. If GH wants to gamify a bit, clicking should open a plot of PDF and a text description like "more/less than <probability> of GitHub population". Everything should work without JavaScript.

**Reply by Nukesor** [reply]
I like this approach. The limits of those achievements also seem kind of arbitrary and they don't offer much value without some context.

In-built statistics however (visibility can of course be toggled), would be super nice. I really like @KOLANICH 's idea on how to introduce some "soft" gamification as well.

**Reply by ThioJoe** [reply]
Perhaps a better achievement would be "merged pull requests from 5 different repositories" or something, so it's more about contributing to multiple projects, not just sheer number of PRs

**Comment by Furkanprlk** [comment]
It seems to me that you should create 2 badges, one to give for a contribution to your repositories, the second for a contribution to someone else's.

**Comment by Maher-Reven** [comment]
well, it would be the case also that people create 2 accounts and do the scenario to validate the badge right?
